### ISSUE TITLE
Cannot save basic auth credentials

### ISSUE THREAD

**Issue by MetroMarv** [keepassxreboot/keepassxc-browser#668]
When accessing a website that requires basic auth, I cannot save the credentials.

## Expected Behavior
A pop up shows up to save the basic auth credentials (like it does when submitting HTML form fields).

## Current Behavior
Nothing happens and no credentials are saved in the password file.

## Steps to Reproduce (for bugs)

1. Unlock the password file using KeePassXC desktop application
2. Open the browser (make sure KeePassXC-Browser is connected to desktop application)
3. Visit a website that requires basic auth (e.g. https://httpbin.org/basic-auth/user/passwd; User: user; Password: passwd)
4. Enter login data and press "OK"

## Debug info

KeePassXC - 2.5.0
KeePassXC-Browser - 1.5.3
Operating system: Linux (Kernel 5.3.7)
Browser: Firefox 70.0 / Chromium 78.0.3904.70

**Comment by phoerious** [comment]
This should probably be moved to the browser repo (@varjolintu?)

**Comment by quicktrick** [comment]
This feature is very much needed.

**Comment by Rain-ctrl** [comment]
Hello everybody,
I see that this issue has been open for two years now and it would be nice to understand why: is it an impossible issue to solve? Is it an economic problem that we could somehow help?

Thank you for this great extension.

**Comment by varjolintu** [comment]
It's not impossible to do. Just a matter of time and priorities. I agree this has been open way too long.

**Comment by varjolintu** [comment]
@futurewebpn Hmm? The automatic filling for HTTP Basic Auth credentials should work just fine. The option is not enabled by default in the extension settings.

**Comment by TOFUFreak** [comment]
Didn't work for me on multiple occasions though, in both Firefox and Chrome (option is enabled in both cases). Sometimes it submits the credentials after canceling the auth dialogue once and reloading the page, sometimes it still doesn't.

**Comment by dharmann** [comment]
> @futurewebpn Hmm? The automatic filling for HTTP Basic Auth credentials should work just fine. The option is not enabled by default in the extension settings.

In my case, I've enabled the option in the chromium extension settings and it asks me for consent, but the Basic Auth form doesn't become filled. Sistematically.

![2022-08-30_14:48:15](https://user-images.githubusercontent.com/6182373/187441154-c4eb9a60-0149-4487-9c42-436c15d3774a.png)

**Comment by futurewebpn** [comment]
> @futurewebpn Hmm? The automatic filling for HTTP Basic Auth credentials should work just fine. The option is not enabled by default in the extension settings.

you are right, that is working with me, at least on the page i need it regulary. Thanks for the hint, seems I didn't read careful enough.
